.storybook/3.COLORS.stories.mdx

```diff
@@ -84,37 +85,35 @@ var(--color-primary-alternative)
 var(--color-primary-muted)
 var(--color-primary-inverse)
 
-var(--color-secondary-default) [DEPRECATED]
-var(--color-secondary-alternative) [DEPRECATED]
-var(--color-secondary-muted) [DEPRECATED]
-var(--color-secondary-inverse) [DEPRECATED]
-var(--color-secondary-disabled) [DEPRECATED]
-
 /** States */
 /** Error */
 var(--color-error-default)
 var(--color-error-default-hover)
 var(--color-error-default-pressed)
 var(--color-error-alternative)
+var(--color-error-default-hover)
+var(--color-error-default-pressed)
```

Comment:
These CSS variables are present and allowed to be used. Updating the docs to reflect this.

<img width="757" alt="Screenshot 2024-06-09 at 8 14 35 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e2d720ef-55d4-4787-a5db-1f298b87661d">